File: v2/jaxrs/src/main/java/io/swagger/model/MemberEvent.java
package io.swagger.model;


import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class MemberEvent  {
  
  private String id = null;
  private String createdAt = null;
  private String type = null;
  private String source = null;
  private String referer = null;
  private String originalUrl = null;
  private String originatorEmail = null;
  private String originatorName = null;
  private String parentOriginatorId = null;
  private String parentEventId = null;
  private String originatorType = null;
  private String originatorGroupId = null;
  private String eventType = null;
  private String eventSource = null;
  private String contentUrl = null;
  private String eventDate = null;
  private String username = null;
  private String memberId = null;
  private String originatorId = null;
  private String subjectId = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("id")
  public String getId() {
    return id;
  }
  public void setId(String id) {
    this.id = id;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("createdAt")
  public String getCreatedAt() {
    return createdAt;
  }
  public void setCreatedAt(String createdAt) {
    this.createdAt = createdAt;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("type")
  public String getType() {
    return type;
  }
  public void setType(String type) {
    this.type = type;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("source")
  public String getSource() {
    return source;
  }
  public void setSource(String source) {
    this.source = source;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("referer")
  public String getReferer() {
    return referer;
  }
  public void setReferer(String referer) {
    this.referer = referer;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("originalUrl")
  public String getOriginalUrl() {
    return originalUrl;
  }
  public void setOriginalUrl(String originalUrl) {
    this.originalUrl = originalUrl;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("originatorEmail")
  public String getOriginatorEmail() {
    return originatorEmail;
  }
  public void setOriginatorEmail(String originatorEmail) {
    this.originatorEmail = originatorEmail;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("originatorName")
  public String getOriginatorName() {
    return originatorName;
  }
  public void setOriginatorName(String originatorName) {
    this.originatorName = originatorName;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("parentOriginatorId")
  public String getParentOriginatorId() {
    return parentOriginatorId;
  }
  public void setParentOriginatorId(String parentOriginatorId) {
    this.parentOriginatorId = parentOriginatorId;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("parentEventId")
  public String getParentEventId() {
    return parentEventId;
  }
  public void setParentEventId(String parentEventId) {
    this.parentEventId = parentEventId;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("originatorType")
  public String getOriginatorType() {
    return originatorType;
  }
  public void setOriginatorType(String originatorType) {
    this.originatorType = originatorType;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("originatorGroupId")
  public String getOriginatorGroupId() {
    return originatorGroupId;
  }
  public void setOriginatorGroupId(String originatorGroupId) {
    this.originatorGroupId = originatorGroupId;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("eventType")
  public String getEventType() {
    return eventType;
  }
  public void setEventType(String eventType) {
    this.eventType = eventType;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("eventSource")
  public String getEventSource() {
    return eventSource;
  }
  public void setEventSource(String eventSource) {
    this.eventSource = eventSource;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("contentUrl")
  public String getContentUrl() {
    return contentUrl;
  }
  public void setContentUrl(String contentUrl) {
    this.contentUrl = contentUrl;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("eventDate")
  public String getEventDate() {
    return eventDate;
  }
  public void setEventDate(String eventDate) {
    this.eventDate = eventDate;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("username")
  public String getUsername() {
    return username;
  }
  public void setUsername(String username) {
    this.username = username;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("memberId")
  public String getMemberId() {
    return memberId;
  }
  public void setMemberId(String memberId) {
    this.memberId = memberId;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("originatorId")
  public String getOriginatorId() {
    return originatorId;
  }
  public void setOriginatorId(String originatorId) {
    this.originatorId = originatorId;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("subjectId")
  public String getSubjectId() {
    return subjectId;
  }
  public void setSubjectId(String subjectId) {
    this.subjectId = subjectId;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class MemberEvent {\n");
    
    sb.append("  id: ").append(id).append("\n");
    sb.append("  createdAt: ").append(createdAt).append("\n");
    sb.append("  type: ").append(type).append("\n");
    sb.append("  source: ").append(source).append("\n");
    sb.append("  referer: ").append(referer).append("\n");
    sb.append("  originalUrl: ").append(originalUrl).append("\n");
    sb.append("  originatorEmail: ").append(originatorEmail).append("\n");
    sb.append("  originatorName: ").append(originatorName).append("\n");
    sb.append("  parentOriginatorId: ").append(parentOriginatorId).append("\n");
    sb.append("  parentEventId: ").append(parentEventId).append("\n");
    sb.append("  originatorType: ").append(originatorType).append("\n");
    sb.append("  originatorGroupId: ").append(originatorGroupId).append("\n");
    sb.append("  eventType: ").append(eventType).append("\n");
    sb.append("  eventSource: ").append(eventSource).append("\n");
    sb.append("  contentUrl: ").append(contentUrl).append("\n");
    sb.append("  eventDate: ").append(eventDate).append("\n");
    sb.append("  username: ").append(username).append("\n");
    sb.append("  memberId: ").append(memberId).append("\n");
    sb.append("  originatorId: ").append(originatorId).append("\n");
    sb.append("  subjectId: ").append(subjectId).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/vestorly/Model/MemberEvent.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace vestorly.model {
  [DataContract]
  public class MemberEvent {
    
    
    [DataMember(Name="_id", EmitDefaultValue=false)]
    public string id { get; set; }

    
    
    [DataMember(Name="created_at", EmitDefaultValue=false)]
    public string createdAt { get; set; }

    
    
    [DataMember(Name="type", EmitDefaultValue=false)]
    public string type { get; set; }

    
    
    [DataMember(Name="source", EmitDefaultValue=false)]
    public string source { get; set; }

    
    
    [DataMember(Name="referer", EmitDefaultValue=false)]
    public string referer { get; set; }

    
    
    [DataMember(Name="original_url", EmitDefaultValue=false)]
    public string originalUrl { get; set; }

    
    
    [DataMember(Name="originator_email", EmitDefaultValue=false)]
    public string originatorEmail { get; set; }

    
    
    [DataMember(Name="originator_name", EmitDefaultValue=false)]
    public string originatorName { get; set; }

    
    
    [DataMember(Name="parent_originator_id", EmitDefaultValue=false)]
    public string parentOriginatorId { get; set; }

    
    
    [DataMember(Name="parent_event_id", EmitDefaultValue=false)]
    public string parentEventId { get; set; }

    
    
    [DataMember(Name="originator_type", EmitDefaultValue=false)]
    public string originatorType { get; set; }

    
    
    [DataMember(Name="originator_group_id", EmitDefaultValue=false)]
    public string originatorGroupId { get; set; }

    
    
    [DataMember(Name="event_type", EmitDefaultValue=false)]
    public string eventType { get; set; }

    
    
    [DataMember(Name="event_source", EmitDefaultValue=false)]
    public string eventSource { get; set; }

    
    
    [DataMember(Name="content_url", EmitDefaultValue=false)]
    public string contentUrl { get; set; }

    
    
    [DataMember(Name="event_date", EmitDefaultValue=false)]
    public string eventDate { get; set; }

    
    
    [DataMember(Name="username", EmitDefaultValue=false)]
    public string username { get; set; }

    
    
    [DataMember(Name="member_id", EmitDefaultValue=false)]
    public string memberId { get; set; }

    
    
    [DataMember(Name="originator_id", EmitDefaultValue=false)]
    public string originatorId { get; set; }

    
    
    [DataMember(Name="subject_id", EmitDefaultValue=false)]
    public string subjectId { get; set; }

    
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MemberEvent {\n");
      
      sb.Append("  id: ").Append(id).Append("\n");
      
      sb.Append("  createdAt: ").Append(createdAt).Append("\n");
      
      sb.Append("  type: ").Append(type).Append("\n");
      
      sb.Append("  source: ").Append(source).Append("\n");
      
      sb.Append("  referer: ").Append(referer).Append("\n");
      
      sb.Append("  originalUrl: ").Append(originalUrl).Append("\n");
      
      sb.Append("  originatorEmail: ").Append(originatorEmail).Append("\n");
      
      sb.Append("  originatorName: ").Append(originatorName).Append("\n");
      
      sb.Append("  parentOriginatorId: ").Append(parentOriginatorId).Append("\n");
      
      sb.Append("  parentEventId: ").Append(parentEventId).Append("\n");
      
      sb.Append("  originatorType: ").Append(originatorType).Append("\n");
      
      sb.Append("  originatorGroupId: ").Append(originatorGroupId).Append("\n");
      
      sb.Append("  eventType: ").Append(eventType).Append("\n");
      
      sb.Append("  eventSource: ").Append(eventSource).Append("\n");
      
      sb.Append("  contentUrl: ").Append(contentUrl).Append("\n");
      
      sb.Append("  eventDate: ").Append(eventDate).Append("\n");
      
      sb.Append("  username: ").Append(username).Append("\n");
      
      sb.Append("  memberId: ").Append(memberId).Append("\n");
      
      sb.Append("  originatorId: ").Append(originatorId).Append("\n");
      
      sb.Append("  subjectId: ").Append(subjectId).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}
